composer.json

```diff
@@ -33,7 +33,7 @@
         "colinmollenhour/credis": "1.16.0",
         "composer/package-versions-deprecated": "1.11.99.5",
         "composer/semver": "3.4.0",
-        "endroid/qr-code": "5.0.9",
+        "endroid/qr-code": "5.0.7",
```

Comment:
Did you mean to roll this back?

Reply:
Certainly not. No idea how I managed to do it without conflicts..